### ISSUE TITLE
Solaris doesn't know "cp -v"

### ISSUE THREAD

**Issue by mojca** [buildbot/buildbot#6870]
After adding
```python
        steps.Git(
            repourl = ...
            method = 'clobber',
            mode = 'full',
            method = 'copy',
        )
```
in an attempt to always do a clean build, Solaris builds broke with:
```
cp: illegal option -- v
Usage: cp [-f] [-i] [-p] [-@] f1 f2
       cp [-f] [-i] [-p] [-@] f1 ... fn d1
       cp -r|-R [-H|-L|-P] [-f] [-i] [-p] [-@] d1 ... dn-1 dn
```

See:
* https://build.contextgarden.net/#/builders/93/builds/1010

Commit that broke the behaviour:
* https://github.com/TeXLive-M/texlive-buildbot/commit/5c2a795825bb93e9e11c59301693edd6a01e3143

**Comment by mokibit** [comment]
Hi, @mojca. Thanks a lot for the bug report. I think I fixed it, could you check this PR  #6880 and let me know if it works now on your system.
I could not test it, since I do not have Solaris.

**Comment by mokibit** [comment]
@p12tic I think this bug can be closed for the time being.
